draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -199,6 +213,11 @@ The following terms are used throughout this document:
   by a cryptographically-secure random number generator.
 - `concat(x0, ..., xN)`: Concatenation of byte strings.  `concat(0x01,
   0x0203, 0x040506) = 0x010203040506`.
+- `split(N1, N2, x)`: Split a byte string `x` of length `N1 + N2` into its first
+  `N1` bytes and its last `N2` bytes.  This function is the inverse of
+  `concat(x1, x2)` when `x1` is `N1` bytes long and `x2` is `N2` bytes long. It
+  is an error to call this function with a byte string that does not have length
+  `N1 + N2`.
```

Comment:
So this is tricky, it's being used to split secret key material and public key material, and being leaky on one is acceptable and being leaky on the other is not...